source/_integrations/bang_olufsen.markdown

```diff
@@ -259,9 +259,10 @@ The Bang & Olufsen integration additionally supports different custom actions
 
 Join a Beolink experience.
 
-| Action data attribute | Optional | Description                           |
-| --------------------- | -------- | ------------------------------------- |
-| `beolink_jid`         | yes      | Manually specify Beolink JID to join. |
+| Action data attribute | Optional | Description                                                                                                                                                                                    |
+| --------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `beolink_jid`         | yes      | Manually specify Beolink JID to join.                                                                                                                                                          |
+| `source_id`           | yes      | Specify which source to join, behavior varies between platforms. Source names prefaced by a platform name can only be used when connecting to that platform. A defined Beolink JID is required |
```

Comment:
Once this [comment is resolved](https://github.com/home-assistant/core/pull/132377#discussion_r1871808853), I'd like to see the possible values that can be supplied enumerated in the documentation.

This is confusing to me:
> Source names prefaced by a platform name can only be used when connecting to that platform.

Is that talking about a HA platform (i.e. `media_player`) or something else? Let's clarify what this is talking about and provide an example.

Reply:
Platform refers to the Bang & Olufsen hardware platform, ASE, Mozart etc.

Reply:
This is about hardware platforms used by Bang & Olufsen. ASE, Mozart and the Beolink converter. This should be clarified.